.github/workflows/linux.yml

```diff
@@ -93,6 +93,8 @@ jobs:
 
       - name: Package
         run: |
+          # Remove the SQL mimer plugin library as we do not have the required library to package within the AppImage
+          rm /home/runner/work/QField/Qt/6.6.0/gcc_64/plugins/sqldrivers/libqsqlmimer.so
```

Comment:
Could we define a env variable `QT_VERSION` on job level and reference to that in `install-qt-action` and in commands like this one?

Reply:
@m-kuhn , good improvement!

Reply:
@m-kuhn , implemented. I didn't use QT_VERSION as I was worried it might conflict with a QT environment variable.